python/cuml/test/dask/test_random_forest.py

```diff
@@ -81,6 +87,24 @@ def test_rf_classification_dask(partitions_per_worker, cluster):
         c.close()
 
 
+def test_rf_throws_exceptions(cluster):
```

Comment:
Could we move this into a more general test for the utility function? Will make it easier to debug as the utility function gets used more widely across the codebase.

Reply:
I kept this RF-specific test but added the more general test in a new test.